react/REACT1/week2/to-do-app/src/todoListComponent.js

```diff
@@ -0,0 +1,60 @@
+import { useState } from "react";
+import TodoItem from "./todoItem";
+
+const todos = [
+  {
+    id: 1,
+    description: "Get out of bed",
+  },
+  {
+    id: 2,
+    description: "Brush teeth",
+  },
+  {
+    id: 3,
+    description: "Eat breakfast",
+  },
+];
+
+export function TodoList() {
+  const [todosState, setTodosState] = useState(todos);
+
+  const addTodo = () => {
+    const newTodo = { description: "learn React", id: todosState.length + 1 };
+    setTodosState((prevTodos) => {
+      return [...prevTodos, newTodo];
+    });
+  };
+
+  const deleteTodo = (id) => {
+    const newTodos = todosState.filter((todo) => {
+      return todo.id !== id;
+    });
+    setTodosState(newTodos);
+  };
+
+  const todoItems = todosState.map((todo) => (
+    <TodoItem
+      description={todo.description}
+      key={todo.id}
+      status={todo.done}
```

Comment:
Hmm, this is a bit confusing, your todo object does not have a 'done' attribute but you're trying to access it here.
See my comment about the done variable in the todoItem component